chainerx_cc/CMakeLists.txt

```diff
@@ -12,49 +12,57 @@ SET(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
 include(cmake/clang-tidy.cmake)
 include(cmake/third-party.cmake)
 
-# Configure options
-option(CHAINERX_BUILD_PYTHON "Build Python binding" OFF)
-option(CHAINERX_BUILD_TEST "Build test" OFF)
-option(CHAINERX_BUILD_EXAMPLES "Build examples" OFF)
-option(CHAINERX_WARNINGS_AS_ERRORS "Make all warnings of compilers into errors" ON)
-option(CHAINERX_ENABLE_THREAD_SANITIZER "Enable thread sanitizer." OFF)
+#---------
+# Options
+#---------
 
 # CHAINERX_CUDNN_USE_CUPY
```

Comment:
Is it possible to make this envvar checking a macro or function?

Reply:
Could be done, but maybe in a separate PR?

Reply:
OK